src/ui.js

```diff
@@ -15,7 +14,9 @@ const ActorDirectory = require("./actors/actor.directory.js");
 // DOM elements
 const clockElem = document.querySelector(".clock");
 const pauseBtn = document.querySelector(".pause-btn");
+const restartBtn = document.querySelector(".restart-btn");
 const dumpStateBtn = document.querySelector(".dumpState-btn");
+const dumpIterationLogBtn = document.querySelector(".dumpIterationLog-btn");
```

Comment:
same here: I think I've removed iteration log dump. maybe it sneaked back in a merge you did?

Reply:
removed